jdaviz/core/template_mixin.py

```diff
@@ -3134,16 +3134,20 @@ def float_precision(column, item):
                 if column in ('slice', 'index'):
                     # stored in astropy table as a float so we can also store nans,
                     # but should display in the UI without any decimals
-                    return int(item)
-                elif column in ('pixel'):
-                    return f"{item:0.03f}"
```

Comment:
@kecnry , I think "0.03f" can be shortened to "0.3f" but maybe I am missing some subtlety here?